packages/perseus-editor/src/widgets/phet-sim-editor.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable @khanacademy/ts-no-error-suppressions */
+/* eslint-disable react/sort-comp */
+import {LabeledTextField} from "@khanacademy/wonder-blocks-form";
+import * as React from "react";
+
+import type {PerseusPhetSimWidgetOptions} from "@khanacademy/perseus";
+
+type DefaultProps = {
+    url: PerseusPhetSimWidgetOptions["url"];
+    description: PerseusPhetSimWidgetOptions["description"];
+};
+
+type Props = DefaultProps & {
+    onChange: (arg1: {
+        url?: Props["url"];
+        description?: Props["description"];
+    }) => void;
+};
+
+class PhetSimEditor extends React.Component<Props> {
+    static defaultProps: DefaultProps = {
+        url: "",
+        description: "",
+    };
+
+    static widgetName = "phet-sim" as const;
+
+    render(): React.ReactNode {
+        return (
+            <div>
+                <LabeledTextField
+                    label={"URL"}
+                    value={this.props.url}
+                    onChange={(url: string) => this.props.onChange({url})}
+                />
+                <br />
```

Comment:
Comment: This might be good to update with a WB spacing component. I think I've seen that more frequently.

Reply:
Thank you! I will make a ticket!